lifelines/fitters/aalen_johansen_fitter.py

```diff
@@ -24,22 +24,23 @@ class AalenJohansenFitter(UnivariateFitter):
     slightly. This will be done automatically and generates a warning.
     """
 
-    def __init__(self, jitter_level=0.0001, seed=None, alpha=0.95):
+    def __init__(self, jitter_level=0.0001, seed=None, alpha=0.95, calculate_variance=True):
```

Comment:
can you add a doc string with the parameters to the above? Similar to https://github.com/CamDavidsonPilon/lifelines/blob/master/lifelines/fitters/coxph_fitter.py#L78-L98